service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -61,6 +79,70 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
         return respond(caseDetails);
     }
 
+    @PostMapping("/submitted")
+    public void handleSubmitted(@RequestBody CallbackRequest callbackRequest) {
+        final CaseData caseData = getCaseData(callbackRequest);
+        CaseDetails caseDetails = callbackRequest.getCaseDetails();
+
+        String migrationId = (String) caseDetails.getData().get(MIGRATION_ID_KEY);
+
+        if ("DFPL-702".equals(migrationId)) {
+            // update supplementary data
+            String caseId = caseData.getId().toString();
+            Map<String, Map<String, Map<String, Object>>> supplementaryData = new HashMap<>();
+            supplementaryData.put("supplementary_data_updates",
+                Map.of("$set", Map.of("HMCTSServiceId", "ABA3")));
+            coreCaseDataApi.submitSupplementaryData(requestData.authorisation(),
+                authToken.generate(), caseId, supplementaryData);
+        }
+        caseDetails.getData().remove(MIGRATION_ID_KEY);
+    }
+
+    private void run702(CaseDetails caseDetails) {
+        CaseData caseData = getCaseData(caseDetails);
+        var caseId = caseData.getId();
+        String caseName = caseData.getCaseName();
+
+        String courtCode = null;
+        if (caseData.getOrders() != null && StringUtils.isNotEmpty(caseData.getOrders().getCourt())) {
+            courtCode = caseData.getOrders().getCourt();
+        } else if (caseData.getCourt() != null) {
+            courtCode = caseData.getCourt().getCode();
+        }
+        if (courtCode == null) {
+            log.warn("Migration {id = DFPL-702, case reference = {}, case state = {}} doesn't have court info "
+                    + "therefore unable to set caseManagementLocation which is mandatory in global search.",
+                caseId, caseData.getState().getValue());
+            return;
```

Comment:
think this should throw our normal Exception so it pops up immediately in the migration tool and is counted